jetty-server/src/main/config/modules/state.mod

```diff
@@ -0,0 +1,19 @@
+# DO NOT EDIT THIS FILE - See: https://eclipse.dev/jetty/documentation/
+
+[description]
+Creates and updates state file used by jetty.sh
+
+[tags]
+start
+
+[depend]
+server
```

Comment:
Ditto. If there is a custom server module, we don't want to pull in the standard server module.

Reply:
The custom module simple needs to say that it provides "server" and then it will work fine.    We have lots of modules that either depend on "server" or are ordered against it, so any attempt to replace `server.mod` with a custom one will need to provide "server" one way or another.

Having said that, the pid module should be before server not depend on it.